react/components/RichTextEditor/index.jsx

```diff
@@ -174,12 +175,12 @@ const RichTextEditor = ({
     label,
     onChange,
     content,
+    canEdit,
```

Comment:
Not a big deal, but in order to maintain consistency in our individual component props of this repo, May be we can rename it to "disabled" prop instead of "canEdit" ?

Reply:
consistency is good. did not pay attention to other components . updated it. thanks for caching it.